composites/SnippetPreview/components/Icon.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+/**
+ * Returns the Icon component.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} Icon component.
+ */
+export const Icon = ( props ) => {
+	const IconComponent = styled( props.icon )`
+		float: left;
+		width: ${ props => props.iconSize }px;
+		height: ${ props => props.iconSize }px;
+		fill: ${ props => props.iconColor };
```

Comment:
In these 3 lines, we're using the props of the styled component. Why not use the props from the parent?